src/elements/form/extras/tiles-input/TilesInput.js

```diff
@@ -33,8 +33,9 @@ export class TilesInput extends BaseInput {
         this.labels = labels;
     }
 
-    onChanged(e) {
-        this.value = e.value ? e.value : null;
```

Comment:
This is a question: do we want the value or the object's value property at this level?